### PR TITLE
Adds a changelog

### PR DESCRIPTION
**Pull request by chuckha**
Signed-off-by: Chuck Ha <chuck@heptio.com>

### CODE REVIEW

**Comment by timothysc** [comment]
@chuckha we removed changelogs in favor of the release pages.

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    - delete      Deletes Kubernetes resources that were generated by a Sonobuoy run
+    - e2e         Inspect e2e test results. Optionally rerun failed tests
+    - gen         Generates a sonobuoy manifest for submission via kubectl
+    - help        Help about any command
+    - logs        Dumps the logs of the currently running sonobuoy containers for diagnostics
+    - retrieve    Retrieves the results of a sonobuoy run to a specified path
+    - run         Submits a sonobuoy run
+    - status      Gets a summarizes status of a sonobuoy run
+    - version     Print sonobuoy version
+ - Secured aggregator <-> worker communication.
```
Comment (liztio) [inline docs/CHANGELOG.md]: maybe mention the preflight checks?

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ - Support for Kubernetes 1.10.
+ - Proper signal handling for graceful plugin  cleanups.
+ - A programmatic interface to Sonobuoy's features.
+ - A Go client to make the results archive easier to programmatically inspect.
+
+### Changed
+ - Support for Kubernetes 1.7 removed.
```
Comment (liztio) [inline docs/CHANGELOG.md]: TLS encryption between client and server

**Comment by chuckha** [comment]
closing in favor of #361

**Pull request closed without merging**